### PR TITLE
Some improvements

### PR DESCRIPTION
**Pull request by martindonat**
I was able to finish what I promised in issue #44 

Have a look there for the description.

### CODE REVIEW

File: gregoriotex/details.html [hunk 1]
```diff
@@ -560,8 +576,8 @@
 <ul>
 <li><code>\gresetlines{invisible}</code> completely removes the staff lines.</li>
 <li><code>\gresetclef{invisible}</code> removes the clefs.</li>
-<li><code>\grechangestaffthicknes{<i>&lt;integer&gt;</i>}</code> changes the thickness of the staff lines. The default value is the same as the staff size. You can only put integers here.</li>
-<li><code>\gresetgregoriofont[op]{parmesan}</code> uses the Dominican varian of the Parmesan font for the neumes.</li>
+<li><code>\grechangestafflinethickness{<i>&lt;integer&gt;</i>}</code> changes the thickness of the staff lines. The default value is the same as the staff size. You can only put integers here.</li>
+<li><code>\gresetgregoriofont[op]{parmesan}</code> uses the Dominican variant of the Parmesan font for the neumes.</li>
```
Comment (rpspringuel) [inline gregoriotex/details.html]: `parmesan` was renammed to `granapadano`

File: gregoriotex/index.html [hunk 2]
```diff
@@ -56,7 +56,7 @@
...
+<p>This page describes quickly the origin and the construction of Gregorio<span class="tex">T<span class="epsilon">e</span>X</span>. You can also take a look at the <a href="./details.html">Gregorio<span class="tex">T<span class="epsilon">e</span>X</span> introduction page</a> or at the <a href="./details.html">Gregorio<span class="tex">T<span class="epsilon">e</span>X</span> detailed description page</a>. If it is the first time you are dealing with <span class="tex">T<span class="epsilon">e</span>X</span>, I advise you to take a look at the <a href="./tex.html"><span class="tex">T<span class="epsilon">e</span>X</span> page</a>.</p>
```
Comment (rpspringuel) [inline gregoriotex/index.html]: You're linking to `details.html` twice.

File: gregoriotex/details.html [hunk 3]
```diff
@@ -158,9 +153,30 @@
...
-	    <td></td>
-	    <td>none (Plain <span class="tex">T<span class="epsilon">e</span>X</span>)</td>
+	    <td><code>annotation</code></td>
+	    <td>the annotation</td>
+	    <td>none</td>
+	</tr>
+	<tr>
+	    <td><code>commentary</code></td>
+	    <td>the commentary</td>
+	    <td><i>footnote-size italics</i></td>
```
Comment (rpspringuel) [inline gregoriotex/details.html]: I haven't checked, but I'm pretty sure that there's a difference in the formatting for PlainTeX here as changing font size is very difficult in PlainTeX and so we don't generally do it.

**Review by rpspringuel — CHANGES_REQUESTED** [review]

File: gregoriotex/details.html (hunk 3 above)
Comment (martindonat) [reply gregoriotex/details.html]: I copied it just from the Reference. See p.30 of the 4.2 Ref

File: gregoriotex/details.html (hunk 1 above)
Comment (martindonat) [reply gregoriotex/details.html]: Ok, I just corrected the command. Will add that

File: gregoriotex/index.html (hunk 2 above)
Comment (martindonat) [reply gregoriotex/index.html]: Right, but there is no GTeX introduction page. So I will just delete the first half of the sentence?

**Comment by martindonat** [comment]
I updated my repository. Do I have to open a new pull request?

**Comment by henryso** [comment]
As long as you make them in the same branch and push that (which you apparently did), that will update the associated (in this case, this) pull request.

File: gregoriotex/details.html (hunk 3 above)
Comment (rpspringuel) [reply gregoriotex/details.html]: That may just mean that the reference manual needs to be corrected too.

File: gregoriotex/details.html [hunk 4]
```diff
@@ -577,7 +577,7 @@
 <li><code>\gresetlines{invisible}</code> completely removes the staff lines.</li>
 <li><code>\gresetclef{invisible}</code> removes the clefs.</li>
 <li><code>\grechangestafflinethickness{<i>&lt;integer&gt;</i>}</code> changes the thickness of the staff lines. The default value is the same as the staff size. You can only put integers here.</li>
-<li><code>\gresetgregoriofont[op]{parmesan}</code> uses the Dominican variant of the Parmesan font for the neumes.</li>
+<li><code>\gresetgregoriofont[op]{granapadano}</code> uses the Dominican variant of the Granapadano font for the neumes.</li>
```
Comment (rpspringuel) [inline gregoriotex/details.html]: I should have been more clear.  The font name is Grana Padano, this becomes `granapadano` for the filename and as the argument of the function.

**Review by rpspringuel — CHANGES_REQUESTED** [review]

File: gregoriotex/details.html (hunk 3 above)
Comment (rpspringuel) [reply gregoriotex/details.html]: I just checked `gregoriotex.tex` (where this style is defined for PlainTeX) and it is as I thought.  I'll see about correcting reference manual.

**Comment by martindonat** [comment]
Now it should ready.

**Review by rpspringuel — APPROVED** [review]

File: gregoriotex/details.html (hunk 3 above)
Comment (rpspringuel) [reply gregoriotex/details.html]: I've fixed the reference manual.  If you could please fix this too.

**Review by rpspringuel — CHANGES_REQUESTED** [review]

**Comment by martindonat** [comment]
Ok?

**Review by rpspringuel — APPROVED** [review]
Looks good.

**Comment by martindonat** [comment]
Sorry, I found another mistake, due to my incompetence regarding Git...

I corrected it in my repo.

**Comment by rpspringuel** [comment]
Since I've merged this PR, you'll need to create another one at this point.

**Pull request merged**